cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -938,6 +938,9 @@ if(davix AND NOT builtin_davix)
 
   if(fail-on-missing)
     find_package(Davix 0.6.4 REQUIRED)
+    if(DAVIX_VERSION VERSION_GREATER_EQUAL 0.6.8 AND DAVIX_VERSION VERSION_LESS 0.7.1)
+      message(WARNING "Davix versions 0.6.8 to 0.7.0 do not work with ROOT, please upgrade to 0.7.1 or later.")
```

Comment:
"do not work with ROOT" sounds like it's ROOT's fault ;-) Maybe "have a severe bug"?

Reply:
How about the middle ground "have a bug and do not work with ROOT"?